server/database.py

```diff
@@ -3,27 +3,34 @@
 import sqlite3
 
 vars = dict()
-
 def create_database_table(database_path):
-    print("Creating database ...")
+    '''
+Function to create a local database given a path
+    :param database_path: path where to create the database
+    :return: True when the process has run correctly
+    '''
```

Comment:
Pas mal ! Mais plusieurs rq basées sur les conventions que j'ai pu rencontrer :

- Les docstrings utilisent plutôt 3 double-apostrophes `"""`.
- On place plutôt la ligne de résumé après les 3 premiers `"""` .
- `Function to` est inutile ici, il suffit de commencer par un verbe à l'infinitif.
- Mettre des points à la fin des lignes.
- Mettre des backticks ` autour des mots ou expressions qui sont du code.

My take :

```python
"""Create a local database given a path.

:param database_path: path where to create the database.
:return: `True` when the process has run correctly.
"""
```